posthog-react-native/src/posthog-rn.ts

```diff
@@ -448,15 +445,14 @@ export class PostHog extends PostHogCore {
     }
   }
 
-  private async captureNativeAppLifecycleEvents(): Promise<void> {
+  private async captureAppLifecycleEvents(): Promise<void> {
     const appBuild = this._appProperties.$app_build
     const appVersion = this._appProperties.$app_version
 
     const isMemoryPersistence = this._persistence === 'memory'
 
-    // version and build are deprecated, but we keep them for compatibility
-    // use $app_version and $app_build instead
-    const properties: PostHogEventProperties = { version: appVersion, build: appBuild }
+    // $app_version and $app_build are already added in the common event properties
```

Comment:
Not sure we need this comment here

Reply:
Since we add the previous version and build, you might think but whats about the current version and build? Well turns out its added elsewhere.

Reply:
Good point, can do